composer.json

```diff
@@ -20,10 +20,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4|^8.0",
+        "php": ">=7.4",
```

Comment:
Is there a gurantee that it works on 8.1?

Reply:
> Is there a gurantee that it works on 8.1?

`^8.0` also includes `8.1`